packages/mds-transactions-service/@types/index.ts

```diff
@@ -91,6 +99,9 @@ export interface TransactionSearchParams {
   provider_id?: UUID
   start_timestamp?: Timestamp
   end_timestamp?: Timestamp
+  before?: string
+  after?: string
+  limit?: number
```

Comment:
`status_type: TRANSACTION_OPERATION_TYPE` should be `status_type: TRANSACTION_STATUS_TYPE`
on line 112.

Github won't let me comment on unchanged lines